src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -117,6 +117,30 @@ R"(#include <stdio.h>
 #include <assert.h>
 #include <complex.h>
 #include <lfortran_intrinsics.h>
+
+char* _lpython_int_to_string(int32_t x) {
+    char s[20];
+    sprintf(s, "%d", x);
+    return s;
+}
+
+char* _lpython_int_to_string(int64_t x) {
+    char s[20];
+    sprintf(s, "%lld", x);
+    return s;
+}
+
+char* _lpython_real_to_string(float x) {
+    char s[20];
+    sprintf(s, "%f", x);
+    return s;
+}
+
+char* _lpython_real_to_string(double x) {
+    char s[20];
+    sprintf(s, "%lf", x);
+    return s;
+}
```

Comment:
I am confused here. These functions are a part of `asr_to_c_cpp.h` and not the generated C code? How will these functions end up there?